website/docs/r/federated_settings_org_config.html.markdown

```diff
@@ -38,12 +39,22 @@ data "mongodbatlas_federated_settings_org_configs" "org_configs_ds" {
 * `post_auth_role_grants` - (Optional) List that contains the default [roles](https://www.mongodb.com/docs/atlas/reference/user-roles/#std-label-organization-roles) granted to users who authenticate through the IdP in a connected organization.
 
 * `domain_restriction_enabled` - (Required) Flag that indicates whether domain restriction is enabled for the connected organization.
-* `identity_provider_id` - (Required) Unique 24-hexadecimal digit string that identifies the federated authentication configuration.
+* `identity_provider_id` - (Required) Legacy 20-hexadecimal digit string that identifies the SAML access identity provider that this connected org config is associated with. This id can be found within the Federation Management UI in Atlas in the Identity Providers tab by clicking the info icon in the IdP ID row of a configured SAML identity provider. Leave as `""` to detach/remove the SAML identity provider.
```

Comment:
FYI: just checked and this can also be retrieved through terraform in `okta_idp_id`

Reply:
Added in b959c8b